11ty/definitions/insane.md

```diff
@@ -1,8 +1,33 @@
 ---
 title: Insane
 slug: insane
-defined: false
+defined: true
+speech: adj
+alt_words:
+  - bizarre
+  - intense
+  - odd
+  - perculiar
+  - weird
 flag:
  level: avoid
  text: ableist language
----
\ No newline at end of file
+reading:
+  - text: Words of the Week – “Insane and Insanity”
+    href: https://therollingexplorer.com/ableist-language-to-avoid-and-acceptable-alternatives-insane-edition/
+---
+
+Insane stems from the Latin word «insanus» for not in healty.
+Similar words are «crazy» or «deranged».
+
+Nowadys, these meanings are associated:
+
+1. in a state of mind which prevents normal perception, behavior, or social interaction; seriously mentally ill.
+2. (of an action or quality) characterized or caused by madness.
+3. extremely annoyed.
+4. extremely foolish; irrational.
+5. shocking; outrageous.
+
+Regardless of how it is used, it is linked to mental illness. Even if you
```

Comment:
It might be helpful to pull in some of the language around the word _crazy_:

>## Issues

>Insane is very commonly used as an adjective to embody a vast array of ideas, often not specifically. It is used so frequently that it sometimes is a filler. Insane can also be used in a derogatory manner for someone with mental or psychiatric disabilities.

>## Impact

>By using ableist language, we are perpetuating violence against people who experience mental or psychological disabilities. Using this language perpetuates those systems and language of harm, regardless of our intent.

>Usage Tip

Be more specific. Typically we can find an alternate definition by taking time to reflect on what emotion we're really feeling.